apps/web/app/(use-page-wrapper)/apps/[slug]/[...pages]/page.tsx

```diff
@@ -36,10 +36,24 @@ export const generateMetadata = async ({ params, searchParams }: PageProps) => {
   );
 };
 
-const getData = withAppDirSsr<GetServerSidePropsResult<any>>(getServerSideProps);
+const getData = withAppDirSsr<any>(getServerSideProps);
 
-export default WithLayout({
-  getLayout,
-  getData,
-  Page: LegacyPage,
-});
+const ServerPage = async ({ params, searchParams }: ServerPageProps) => {
+  const p = paramsSchema.safeParse(params);
+  if (!p.success) {
+    return notFound();
+  }
+
+  const context = buildLegacyCtx(headers(), cookies(), params, searchParams);
+  const props = await getData(context);
+  if (props.appName === "routing-forms") {
+    return (
+      <Shell backPath="/apps/routing-forms/forms" withoutMain withoutSeo>
```

Comment:
this replaces all the `getLayout`s I removed from routing-form client components below:

https://github.com/calcom/cal.com/pull/18956/files#r1945890875